docs/architecture/deployment/private-saas/download-logs.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Download Logs
+id: download-logs
+description: Download Prophecy support logs
+sidebar_position: 3
+tags:
```

Comment:
we have some pre-requisite for enabling this feature for private SAAS: documented here : https://coda.io/d/Cloud_dptsTdZM0LD/Setting-up-prophecy-log-download-options_suDxgcHw#_luDrg_xx

Reply:
Done.